### PR TITLE
travis: fix git complaining about 'ambiguous argument'

### PR DESCRIPTION
**Pull request by cgundogan**
Fixes the travis bug that was discussed in https://github.com/RIOT-OS/RIOT/pull/4386#issuecomment-162347903

### CODE REVIEW

**Comment by OlegHahm** [comment]
I don't quite get the bug or the fix, but if Travis runs again, I'm fine. ACK

**Comment by cgundogan** [comment]
I read on some SO thread that it might be depending on the shell's prompt. If it includes some sort of git information

**Comment by cgundogan** [comment]
I also deleted the `git log -1 --pretty=format:%H riot/master` line if it's okay with you?
I never saw the output of this command in the travis log .. and I think this information is quite useless.

File: .travis.yml [hunk 1]
```diff
@@ -30,9 +30,7 @@ install:
     - git config --global user.email "travis@example.com"
     - git config --global user.name "Travis CI"
 
-    - git remote add riot https://github.com/RIOT-OS/RIOT.git
-    - git fetch riot master
-    - git log -1 --pretty=format:%H riot/master
+    - git clone -b master https://github.com/RIOT-OS/RIOT.git
```
Comment (cgundogan) [inline .travis.yml]: I think this line is pretty useless and also not right. I will delete this. Should work without it.

**Comment by cgundogan** [comment]
@OlegHahm please re-ACK

**Comment by OlegHahm** [comment]
Where is RIOT cloned now?

**Comment by cgundogan** [comment]
nowhere, because it already runs within the RIOT folder (where the .travis file is hosted)

**Comment by OlegHahm** [comment]
Ah, okay. ACK (but somehow the M0 build constantly fails or gets cancelled)

**Comment by cgundogan** [comment]
Hmm, it's still running for now. Let's wait for the result.
I am not sure if this PR is the right solution, because the shell scripts that get invoked are doing rebasing stuff on the riot master and so forth. It has no effect for our riot repository, but it will behave differently for forked repos of riot (it will not rebase on the riot/master anymore)

**Comment by cgundogan** [comment]
But then again, I am not sure if rebasing a forked repo on the riot/master was a good choice in the first place

**Comment by OlegHahm** [comment]
Travis ran into a timeout for Cortex-M0 3 times. Annoying. I'm willing to give it one more try and then merge without. I can think of no reason why this should not work for a particular platform.

**Comment by OlegHahm** [comment]
haha - deadlock situation. This PR won't pass Travis without #4421 and #4421 won't pass without this one.

Let's merge this despite of Travis. If this breaks something I will spend some beers.

**Comment by DipSwitch** [comment]
Beer! \o/ But hopefully nothing broke =)

**Pull request merged**